### DISCUSSION TITLE
Gradient from/to inherit seemingly does nothing

### DISCUSSION THREAD

**Discussion by JamesManningR** [tailwindlabs/tailwindcss#10696] [Help]
This is in somewhat theory, but {from,to}-inherit doesn't do anything.

Am I right in thinking that it's impossible for this value to actually work within the context of color stops?

I was hoping to create a dynamic page effect with the following classes
`before:absolute before:top-0 before:left-0 before:z-20 before:block before:h-full before:w-4 before:bg-gradient-to-r before:from-inherit before:to-transparent` etc

**Comment by wongjn** [comment]
It would work, since it would inherit down the `--tw-gradient-from` CSS property that Tailwind uses behind the scenes, [example Tailwind Play](https://play.tailwindcss.com/TJDhWDrp35).

**Reply by JamesManningR** [reply]
Ah, that makes sense. So in this situation you would: 
set the parents from and to to the same colour

And inherit one of the two in the pseudo element.

Thanks for clearing that up!
@wongjn
